src/main/groovy/com/bmuschko/gradle/clover/InstrumentCodeAction.groovy

```diff
@@ -95,7 +95,13 @@ class InstrumentCodeAction implements Action<Task> {
                 }
 
                 getMethodContexts().each {
-                    ant.methodContext(name: it.name, regexp: it.regexp)
+                    ant.methodContext(
+                        name:   it.name,
+                        regexp: it.regexp,
+                        maxComplexity: it.maxComplexity,
+                        maxAggregatedComplexity: it.maxAggregatedComplexity,
+                        maxAggregatedStatements: it.maxAggregatedStatements
```

Comment:
It won't, see this stacktrace.  

    Caused by: : Can't assign value 'null' to attribute maxaggregatedcomplexity, reason: class     java.lang.NumberFormatException with message 'For input string: "null"'
	at org.apache.tools.ant.IntrospectionHelper$11.set(IntrospectionHelper.java:1200)
	at org.apache.tools.ant.IntrospectionHelper$AttributeSetter.setObject(IntrospectionHelper.java:1493)
	at org.apache.tools.ant.IntrospectionHelper.setAttribute(IntrospectionHelper.java:411)
	at org.apache.tools.ant.RuntimeConfigurable.maybeConfigure(RuntimeConfigurable.java:528)
	at org.apache.tools.ant.RuntimeConfigurable.maybeConfigure(RuntimeConfigurable.java:464)

Will need to cleanly build a map with non-null keys.